### ISSUE TITLE
new version

### ISSUE THREAD

**Issue by vysheng** [vysheng/tg#476]
Hope will write new version with support for new features till end of next week. Sorry that I can't answer to all these issues. I'll read them after I end new version, since there are many changes.

**Comment by jonnywilliamson** [comment]
@vysheng - I'd just like to publically thank you for continuing to work on this project. It is really really appreciated!

**Comment by SyRenity** [comment]
Thanks for your work and looking forward to the new version!

**Comment by antonioacsj** [comment]
@vysheng <https://github.com/vysheng>, congrats for your great jog in this
project. It has openned many doors in a counterless apps.

Thanks.

2015-04-15 8:05 GMT-03:00 SyRenity <notifications@github.com>:

**Comment by schossel** [comment]
@vysheng I was really afraid that you've abandoned this project. This would be really sad because it's one of the coolest features I've running on my Pi. Please keep up the good work!

**Comment by LukeLR** [comment]
Yes, I totally agree! I'm using Telegram-cli on my Pi as well, and it controls my sockets by using a custom version of RCSwitch.cpp, as well as a RGB-LED-Strip, digitally addressable of course, all by sending commands over telegram. This is so nice and fun, please keep telegram-cli going! Thank you so much for this great application!

**Comment by luckydonald** [comment]
Wuhu! Awesome!

**Comment by nullifye** [comment]
great work and looking forward the new version... keep it up!
